bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java

```diff
@@ -44,29 +51,33 @@
 public class HBaseRequestAdapter {
 
   public static class MutationAdapters {
+    private final boolean allowServersideTimestamp;
```

Comment:
I thought we were going to use a private getter on PutAdapter? I would like to avoid having parallel flags that have to be kept in sync

Reply:
Sorry I misread your reply when you mentioned just private getter instead of package private getter. Also didnt notice this reply either. Updated the PR.